### PR TITLE
Updating README.md file

### PR DESCRIPTION
**Pull request by smandaRH**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] Optimization
- [X] Documentation Update

## Description
Updates in README.md file

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Review by krishvoor — APPROVED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -70,15 +70,24 @@ rm -rf bin rendered tmp templates/grafonnet-lib
...
-- Etcd Dashboard
-- OpenShift Performance Dashboard
-- OVN Dashboard
-- Pgbench Dashboard
-- UPerf Results Dashboard
-- Vegeta Dashboard
-- YCSB Dashboard
+ 
+- CPT
+    - [x] Ingress Perf Dashboard.
```
Comment (vishnuchalla) [inline README.md:75]: can we just have checkboxes here? when we do a preview, a bullet point and then a checkbox appears which doesn't look that good.

Comment (smandaRH) [reply README.md:75]: actually, this looks in this format:
image

**Review by vishnuchalla — COMMENTED** [review]
lgtm

**Review by vishnuchalla — APPROVED** [review]

**Pull request merged**